numpy/lib/twodim_base.py

```diff
@@ -153,6 +153,9 @@ def eye(N, M=None, k=0, dtype=float):
       to a lower diagonal.
     dtype : data-type, optional
       Data-type of the returned array.
+    order : {'C', 'F'}, optional
+        Whether the output should be stored in row-major (C-style) or
+        column-major (Fortran-style) order in memory.
```

Comment:
The versionadded :: tag would be good (grep the code for examples).

Reply:
Ah, also, please format the commit message according to our contribution guide, such as "ENH: ..."

Reply:
(Then again, I guess nowadays, we can also do that during merging in the squashing, didn't really get used to that yet, heh)

Reply:
1. I said `1.13.4` but feel like that isn't quite the right version?
1. I just read the [commit guide][1], thanks!

[1]: https://docs.scipy.org/doc/numpy-dev/dev/gitwash/development_workflow.html#writing-the-commit-message